### DISCUSSION TITLE
Optional Thinking Cap

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/22S-ASIAAM-191A#24] [Extra Credit]
# Extra Thinking Cap

### Due 4/25

>    ## How can local communities and take ownership of technology and surveys related to themselves?

Reflect on how we are starting to think about collecting data, think about what it means to empower local communities and make the tools their own.

#### Note
Because this is an optional thinking cap, commenting on other's post is not required, but those that do will get extra credit for doing so.

**Comment by antonyblin** [comment]
Local communities can take ownership of technology and surveys related to themselves by ensuring that data that is collected about them is also made available to them, such that the information is not just harvested and lost within the realm of academia, never to be seen again or made use of. Investing time and resources into assisting with data collection without being able to reap the benefits in some form makes for a disempowered community. Additionally, community action and outreach encourages community members to stand up for themselves and be active members of the data collection process, lessening opportunities for researchers and outside entities to exploit their community (whether intentionally or not).

**Reply by codingdianat** [reply]
I agree that the communities should have access to the data that is collected from them. In addition, I think that the community should also know what the data is use for. This helps the community be aware of whether the data that is collected will be applied in a manner that the communities accept.

**Reply by angelineeow** [reply]
Hi,
I like that you mentioned the local communities should have access to the data collected as the data should not be lost within the realm of academia! I also like you said that community action and outreach encourage community members to stand up for themselves. I would definitely agree with that part as people will only learn how to stand up for themselves with encouragement. Great job!

**Comment by codingdianat** [comment]
Local communities can take ownership of technology and surveys related to themselves by being proactive about how outside influences may impact them. Being aware of the benefits and downsides of certain technologies can cultivate more responsible usage. This can result in technology that communities can use to uplift themselves rather than depend on outsiders. Being knowledgeable about certain technologies allows communities to not be dependent on outsiders who may have conflicting personal objectives. One personal objective may be their desire to collect survey data on the community. Local communities should consider whether the questions they are being asked are meant to truly understand them or to confirm the bias of outsiders. Being able to recognize when they are disrespected and misrepresented allows communities to better advocate for themselves during the surveying process.

**Reply by angelineeow** [reply]
Hi,
I agree that local communities can be proactive about how outside influences may impact them to take ownership of technology and surveys related to themselves. I really like you mentioned that with this, the technology can be used to uplift the local communities themselves rather than outsiders! Great work!

**Comment by angelineeow** [comment]
Local communities can take ownership of technology and surveys related to themselves by paying attention to the needs of their communities. Local communities being attentive to their own surrounding would fully benefit from the ownership of technology and surveys related to themselves as they would be the right member who has similar experience and the ability to listen to others in their community to provide community support. Having the right people taking ownership of technology and surveys related to the local communities would empower the local communities directly and increase the practicality of tools created. Besides that, local communities can also take ownership of technology and surveys related to themselves when they have the access to the data collection when the tools are created. This helps in showing respect to the local communities by being open with what the data is being used after data is collected.

**Reply by XiongHer** [reply]
Hi, I like your point about the showing respect and having the right people to work with local communities to increase practical tools and resources. Local people have great knowledge and resources themselves, so it is important to listen and incorporate what they have to the overall project. Also, it is their information that researchers are collected, so it is important to ask for permission about how this information is being used and how it will benefit and improve both parties rather than just the researchers' goals/objectives. Also, collaboration and listening to community will reduce the power dynamic and strengthen their relationship for future works to continue sharing knowledge.

**Comment by XiongHer** [comment]
Participatory action research (PAR) is a good approach to empower local communities as well as give them the ownership of technologies (knowledge) and surveys. This approach include local community members in the whole research/project process and work together side by side with researchers to better understand certain issues/problems that they faced and how to go about changing it. It is about listening to the local community's voices and concerns and what they want to improve rather being told by researchers what to do/change. As a result, local members have the chance to sit in the decision-making table and understand what researchers wrote or said about them. PAR also enhance both parties' relationship and trust, providing local communities with tools and resources to continue the work after the research/project is concluded. Moreover, it is important to protect both individual and community's privacy, keeping information confidential and ensuring the safety of both the individual and the community.

**Reply by codingdianat** [reply]
I like your idea of having community members be involved in the research/project process. I think that having community members be involved with researchers will limit the potential biases that researchers may have. This will allow the process to be advantageous to both parties involved.

**Reply by kris10fu** [reply]
Hi! I really like that PAR incorporates local community members as the owners of the entire process rather than outside researchers coming in and taking over. I think that allowing local communities to be the leader in the decision-making process allows for their needs and desires to be at the center. I also agree that ensuring privacy is really important because it can not only prevent exploitation and other harmful practices but can also improve the trust and respect between local communities and outsiders.

**Comment by joonyloony** [comment]
I think that local communities can take ownership of technology and surveys related to them through getting more involved in the very surveys that are being created to learn/gather data from those communities. From a smaller scale, nonprofits and other smaller organizations as well as individual surveyors should make a larger effort in ethically collecting data through directly involving the community members as well as respecting their time and privacy. Though, there is only so much that one can do when it is a ruling/powerful entity collecting the data and it is at their discretion how much data they can share with the general public as well as how involved the community members can become. I think that in order to empower local communities to make the tools their own, there should be larger emphasis placed on open source tools as well as workshops to help locals get involved. We can already see that “​​Social and political pressures are continually changing the boundary over what types of individual-level data collected by government should be made publicly available,” and so we should continue to push and advocate for transparency as well as legislation that may require private entities to do the same.

**Reply by kris10fu** [reply]
Hi! I really agree with your point about involving the local community in the ethical collection of data. I think that providing workshops and educating them more about how to get involved is really important because there are often not a lot of opportunities regarding this. Being more educated about the ownership of technology and surveys can empower local communities to know what to look out for when partnering with outsiders, especially in terms of exploitative activities.

**Comment by kris10fu** [comment]
Local communities can take ownership of technology and surveys related to themselves by making sure that the tools that are being used empower and respects the community members. We’ve talked about the ethics of involving local communities in projects, such as participatory mapping, and the many pitfalls that can occur if there are no proper guidelines as to how to empower the members to become actively involved in the process. Empowerment of community members can come in many forms, but encouraging active participation, such as allowing them to share their stories on surveys, in the ownership of technology and surveys related to them is a good starting point. This would help reduce biases in the actual data collection and analysis process. Advocating for the needs of local communities through making sure that members are respected or listened to by outsiders that are coming in can be important for taking ownership of whatever data or technology comes out as end products. Ensuring that they own the data or surveys that are related to the local community is also crucial for making sure that there is no exploitation from outsiders, such as large corporations.
